packages/ui/src/working-groups/queries/workingGroups.graphql

```diff
@@ -49,7 +49,7 @@ fragment WorkingGroupFields on WorkingGroup {
     ...WorkingGroupMetdataFields
   }
   workers {
-    id
+    stake
```

Comment:
So we don't use workers ids anywhwere? IIRC we changed how the list of workers is fetched, so this makes sense. Just to double check.

Reply:
Yep, we catch workers separately from working group now.
That's why having worker ids is useless now.